prisma/migrations/20240416013657_add_worker_rating/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "ShiftAssignment" ADD COLUMN     "rating" INTEGER;
```

Comment:
- Consider adding a CHECK constraint to ensure rating is between 1 and 5:
`ALTER TABLE "ShiftAssignment" ADD CONSTRAINT "valid_rating" CHECK (rating >= 1 AND rating <= 5);`
- Consider adding a "ratedAt" timestamp column to track when ratings were submitted.

Reply:
@bikash119 adding CONSTRAINT is not required here. as it can be null for already existing shifts. 

adding ratedAt column also kind of unnecessary. subject to discussion